feature/guia nuevas practicas (#1525)
mercadopago/devsite-docs: guides/best-practices/ux-for-checkouts/measure-and-optimize.pt.md

# Meça e otimize seus resultados

> Para saber sobre o desempenho do seu checkout depois de implementar essas boas práticas, recomendamos que você integre alguma ferramenta de acompanhamento (por exemplo, o Google Analytics) e faça otimizações com base nos resultados para melhorar a experiência dos seus usuários e ter mais conversão.

Reviewer: ```suggestion
Para avaliar o desempenho do seu checkout após implementar essas boas práticas, recomendamos a integração de alguma ferramenta de acompanhamento (por exemplo, o Google Analytics). Essa ferramenta trará dados importantes que servirão como base para realizar otimizações, melhorar a experiência dos seus usuários e ter mais conversão.
```

![pt Medir y optimizar](/images/best-practices-guide/PortMedirOptimizar.png)

> Agora, você conta com todas as informações de que precisa para criar a melhor experiência de pagamento para seus clientes. Integre nossos produtos [Checkout API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/introduction) ou as soluções de [Plugins e plataformas](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/plugins) e otimize a experiência do usuário com essas boas práticas.

Reviewer: ```suggestion
Agora, você conta com todas as informações necessárias para criar a melhor experiência de pagamento para seus clientes. Integre nossos produtos [Checkout API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/introduction) ou as soluções de [Plugins e plataformas](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/plugins) e otimize a experiência do usuário com essas boas práticas.
```

> Agora, você conta com todas as informações necessárias para criar a melhor experiência de pagamento para seus clientes. Integre nossos produtos [Checkout API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-api/introduction) ou as soluções de [Plugins e plataformas](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/plugins) e otimize a experiência do usuário com essas boas práticas.

Author: ```suggestion
Agora, você conta com todas as informações necessárias para criar a melhor experiência de pagamento para seus clientes. Integre nossos produtos e otimize a experiência do usuário com essas boas práticas.
```

> PREV_STEP_CARD_PT
>
> Crie um checkout ágil
>

> > Siga estas etapas para criar um checkout fácil de navegar e entender.

Reviewer: ```suggestion
> Saiba como criar um checkout com navegação simples e de fácil compreensão.
```
